workspaces/marketplace/.changeset/calm-dolls-laugh.md

```diff
@@ -0,0 +1,5 @@
+---
+'@red-hat-developer-hub/backstage-plugin-marketplace': patch
```

Comment:
Probably should use `minor` here for a new feature?

Reply:
Yeah, I wasn't sure since the feature stays disabled until the API is in place